python/dgl/backend/backend.py

```diff
@@ -398,6 +561,41 @@ def gather_row(data, row_index):
     """
     pass
 
+def slice_axis(data, axis, begin, end):
```

Comment:
We have `narrow_row` for this purpose but only on the first axis. Should we refactor the `narrow_row` occurrences to this interface?

Reply:
I think it's ok to preserve the two APIs.

Reply:
Okay.